Zotero/Scenes/Detail/PDF/Views/Annotation View/AnnotationViewHeader.swift

```diff
@@ -178,27 +178,32 @@ final class AnnotationViewHeader: UIView {
         authorLabel.setContentHuggingPriority(.defaultLow, for: .horizontal)
         authorLabel.translatesAutoresizingMaskIntoConstraints = false
 
+        var shareConfig = UIButton.Configuration.plain()
+        shareConfig.image = UIImage(systemName: "square.and.arrow.up")
+        shareConfig.contentInsets = NSDirectionalEdgeInsets(top: 0, leading: layout.horizontalInset, bottom: 0, trailing: (layout.horizontalInset / 2))
         let shareButton = UIButton()
-        shareButton.setImage(UIImage(systemName: "square.and.arrow.up"), for: .normal)
+        shareButton.configuration = shareConfig
         shareButton.tintColor = Asset.Colors.zoteroBlueWithDarkMode.color
-        shareButton.contentEdgeInsets = UIEdgeInsets(top: 0, left: layout.horizontalInset, bottom: 0, right: (layout.horizontalInset / 2))
         shareButton.setContentCompressionResistancePriority(.required, for: .horizontal)
         shareButton.setContentHuggingPriority(.defaultHigh, for: .horizontal)
-        
+
+        var menuConfig = UIButton.Configuration.plain()
```

Comment:
`menuConfig` is not assigned to the button below.

Reply:
Good catch, fixed in 95b644e50fb980fbd2b87c67de69c6e0767dadf9